spec/controllers/testing_grounds/caching_spec.rb

```diff
@@ -43,10 +44,10 @@
     let(:opts) { { strategies: {} } }
 
     it 'caches the data request' do
-      get :data, id: testing_ground
+      post :data, id: testing_ground
 
       loads = network.node('CONGESTED_END_POINT_0').get(:load)
-      expect(loads.length).to eq(8760)
+      #expect(loads.length).to eq(8760)
```

Comment:
This is something I did not intent to commit at all. I'm thinking this accidentally got in.